src/debug.c

```diff
@@ -2127,6 +2320,61 @@ void BufferExpansionVersion(struct ScriptContext *ctx)
         string = StringCopy(string, sText_Unreleased);
 }
 
+static void DebugAction_TimeMenu_PrintTime(u8 taskId)
+{
+    Debug_DestroyMenu_Full(taskId);
+    LockPlayerFieldControls();
+    Debug_DestroyMenu_Full_Script(taskId, Debug_EventScript_TellTheTime);
+}
+
+void DebugMenu_CalculateTime(struct ScriptContext *ctx)
+{
+    if (OW_USE_FAKE_RTC == TRUE)
```

Comment:
```suggestion
    if (OW_USE_FAKE_RTC)
```

This is on me, I should have noticed this the previous time, sorry